### PR TITLE
Add retrieval of brokerage information

### PR DESCRIPTION
**Pull request by mdusher**
Adding the ability to retrieve brokerage information.

### CODE REVIEW

File: stake/order.py [hunk 1]
```diff
@@ -40,6 +40,16 @@ class Config:
         alias_generator = camelcase
 
 
+class Brokerage(BaseModel):
```
Comment (stabacco) [inline stake/order.py:43]: Could the above brokerage class from the order module be reused in this case?

**Review by stabacco — COMMENTED** [review]
Thanks @mdusher for taking the time to adding this feature. It all looks good to me. The only question is whether it would make sense to reuse the same Brokerage class.

File: stake/order.py (hunk 1 above)
Comment (stabacco) [reply stake/order.py:43]: actually scratch that, i have now remembered that the ASX module is mostly a copy of this

**Review by stabacco — COMMENTED** [review]
Looks good to me, I will try to fix the tests before merging this.

**Comment by stabacco** [comment]
managed to get the tests running, can you please merge master in so that we give it another try?

**Comment by mdusher** [comment]
I've pulled it in and fixed the linting that pre-commit raised. Looks like it's only failing the tests that require a session token now.

**Review by stabacco — APPROVED** [review]

**Pull request merged**